DQM/L1TMonitor/src/L1TStage2BMTF.cc

```diff
@@ -35,70 +36,114 @@ void L1TStage2BMTF::beginLuminosityBlock(const edm::LuminosityBlock& iLumi, cons
 
 void L1TStage2BMTF::bookHistograms(DQMStore::IBooker &ibooker, const edm::Run& iRun, const edm::EventSetup& eveSetup)
 {
+  std::string histoPrefix = "bmtf";
+  if (kalman) {
+    histoPrefix = "kbmtf";
+  }
+
   ibooker.setCurrentFolder(monitorDir);
-  
-  bmtf_hwEta = ibooker.book1D("bmtf_hwEta", "HW #eta", 461, -230.5, 230.5);
-  bmtf_hwLocalPhi = ibooker.book1D("bmtf_hwLocalPhi", "HW Local #phi", 201, -100.5, 100.5);
-  bmtf_hwGlobalPhi = ibooker.book1D("bmtf_hwGlobalPhi", "HW Global #phi", 576, -0.5, 575.5);
-  bmtf_hwPt  = ibooker.book1D("bmtf_hwPt", "HW p_{T}", 512, -0.5, 511.5);
-  bmtf_hwQual= ibooker.book1D("bmtf_hwQual" , "HW Quality", 20, -0.5, 19.5);
-  bmtf_proc  = ibooker.book1D("bmtf_proc" , "Processor", 12, -0.5, 11.5);
-
-  bmtf_wedge_bx  = ibooker.book2D("bmtf_wedge_bx", "Wedge vs BX", 12, -0.5, 11.5, 5, -2.5, 2.5);
+  bmtf_hwEta = ibooker.book1D(histoPrefix+"_hwEta", "HW #eta", 461, -230.5, 230.5);
+  bmtf_hwLocalPhi = ibooker.book1D(histoPrefix+"_hwLocalPhi", "HW Local #phi", 201, -100.5, 100.5);
+  bmtf_hwGlobalPhi = ibooker.book1D(histoPrefix+"_hwGlobalPhi", "HW Global #phi", 576, -0.5, 575.5);
+  bmtf_hwPt  = ibooker.book1D(histoPrefix+"_hwPt", "HW p_{T}", 512, -0.5, 511.5);
+  bmtf_hwQual= ibooker.book1D(histoPrefix+"_hwQual" , "HW Quality", 20, -0.5, 19.5);
+  bmtf_proc  = ibooker.book1D(histoPrefix+"_proc" , "Processor", 12, -0.5, 11.5);
+
+  bmtf_wedge_bx  = ibooker.book2D(histoPrefix+"_wedge_bx", "Wedge vs BX", 12, -0.5, 11.5, 5, -2.5, 2.5);
   bmtf_wedge_bx->setTitle(";Wedge; BX");
   for (int bin = 1; bin < 13; ++bin) {
     bmtf_wedge_bx->setBinLabel(bin, std::to_string(bin), 1);
   }
 
-  bmtf_hwEta_hwLocalPhi = ibooker.book2D("bmtf_hwEta_hwLocalPhi", "HW #eta vs HW Local #phi" , 461, -230.5, 230.5, 201, -100.5, 100.5);
+  bmtf_hwEta_hwLocalPhi = ibooker.book2D(histoPrefix+"_hwEta_hwLocalPhi", "HW #eta vs HW Local #phi" , 461, -230.5, 230.5, 201, -100.5, 100.5);
```

Comment:
Remove the setTitle calles and change the booking calls like
`bmtf_hwEta_hwLocalPhi = ibooker.book2D(histoPrefix+"_hwEta_hwLocalPhi", "HW #eta vs HW Local #phi;HW #eta;HW Local #phi" , 461, -230.5, 230.5, 201, -100.5, 100.5);`

Reply:
@dtsitson please consider also this comment. It would improve the display of the histograms.